Packs/Anomali_ThreatStream/Integrations/Anomali_ThreatStream_v2/Anomali_ThreatStream_v2.py

```diff
@@ -892,18 +892,29 @@ def get_indicators(**kwargs):
         Returns filtered indicators by parameters from ThreatStream.
         By default the limit of indicators result is set to 20.
     """
+    limit = kwargs['limit'] = int(kwargs.get('limit', 20))
+    offset = kwargs['offset'] = 0
     if 'query' in kwargs:
-        params = build_params(q=kwargs['query'], limit=kwargs.get('limit', 20))
-    else:
-        params = build_params(**kwargs)
-
+        kwargs['q'] = kwargs['query']
+        kwargs.pop('query', None)
+    params = build_params(**kwargs)
     iocs_list = http_request("GET", "v2/intelligence/", params=params).get('objects', None)
-
     if not iocs_list:
         demisto.results('No indicators found from ThreatStream')
         sys.exit()
-
     iocs_context = parse_indicators_list(iocs_list)
+    if limit > 1000:
```

Comment:
not sure how are you dealing with the first request here, it should be up to 1000 indicators, no?

Reply:
Between lines 901 and 905 we handle a standard user request (which is usually less than 1000)
If the request is for more than 1000 then we execute this request again and again each time with an offset of the indicators we have already received until reaching the number the user wanted.

Reply:
shouldn't we update the limit to 1000 in that case?